src/Courses/CourseCreation/TopicCreationModal.tsx

```diff
@@ -373,7 +373,7 @@ export const TopicCreationModal: React.FC<TopicCreationModalProps> = ({ unitInde
         });
         console.log(problemsWithOrdering);
```

Comment:
Not in this pr but still 😈

Reply:
In my defense, this Modal is now deprecated!